api/src/main/java/org/openmrs/module/referencemetadata/AdministrativeNotification.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.referencemetadata;
+
+import java.util.Collection;
+
+import org.openmrs.Concept;
+import org.openmrs.api.context.Context;
+
+public class AdministrativeNotification {
```

Comment:
Has this class got anything to do with the ticket?

Reply:
well i was just trying to figure out , how i can display an administrative notification on the user UI .. but i failed to get exactly an example of how to use the  org.openmrs.module.appframework.factory.AdministrativeNotificationProduce

Reply:
How is it related to this ticket?

Reply:
i thought that method can be used to check whether duplicate concepts exists , and then trigers the administration Notification requested by he ticket

Reply:
though am not yet sure how to display an administration notification on the UI as done in the atlas module. i failed to gt any reference documentation on how to do it

Reply:
I'm sure this class is redundant. What is that inevitable usecase for introducing this class?

Reply:
oh ya samwel , acoording to the acceptance criteria  ...
*Upon upgrading an existing OpenMRS installation, if it has two TRUE or FALSE concepts, the administrator should be shown instructions about how to clean this up in their database.*

Reply:
soi was trying out making a method that will trigger the amin notification

Reply:
but am soon pushing , i have modified that

Reply:
hi dan , let me close this PR, i have created a new one

Reply:
please look at this   https://github.com/openmrs/openmrs-module-referencemetadata/pull/40

and https://github.com/openmrs/openmrs-module-referenceapplication/pull/50

am closing this